src/__tests__/e2e.spec.ts

```diff
@@ -39,7 +39,7 @@ describe('Price Estimation Server', () => {
       expect(estimate.baseTokenId).toBe('1')
       expect(estimate.quoteTokenId).toBe('7')
       expect(estimate.buyAmountInBase).toBeGreaterThan(0)
-      expect(estimate.sellAmountInQuote).toBe(1000000000000000000)
+      expect(estimate.sellAmountInQuote).toBe('1000000000000000000')
```

Comment:
Single quotes? Do we use a different linter than all other J/TS projects?

Reply:
The linter tells me to use single quotes so I use them.

Reply:
>  Do we use a different linter than all other J/TS projects?

This uses [`eslint-config-standard` plugin for StandardJS](https://standardjs.com/) style.